packages/core/src/App/Containers/OneTimeDepositModal/one-time-deposit-modal.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { useState } from 'react';
+import {
+    Button,
+    DesktopWrapper,
+    Icon,
+    InlineMessage,
+    Loading,
+    MobileFullPageModal,
+    MobileWrapper,
+    Modal,
+    Text,
+} from '@deriv/components';
+import { useDepositFiatAddress } from '@deriv/hooks';
+import { isMobile } from '@deriv/shared';
+import { observer, useStore } from '@deriv/stores';
+import { Localize, localize } from '@deriv/translations';
+import useLiveChat from 'App/Components/Elements/LiveChat/use-livechat';
+
+const OneTimeDepositModal = () => {
+    const { data: iframe_url, isSuccess } = useDepositFiatAddress();
+    const [is_iframe_loading, setIsIframeLoading] = useState(true);
+    const { client, ui } = useStore();
+    const { has_deposited_for_first_time, loginid } = client;
```

Comment:
has_deposited_for_first_time was just created for testing purpose. we need to find a better way to trigger this modal